rules/private/phases/phase_coda.bzl

```diff
@@ -5,12 +5,4 @@
 #
 
 def phase_coda(ctx, g):
-    dynamic = {}
-    if hasattr(g, "coverage"):
-        dynamic["instrumented_files"] = g.coverage.instrumented_files
-
-    return struct(
-        java = g.ijinfo.intellij_info,
-        providers = g.out.providers,
-        **dynamic
-    )
+    return g.out.providers
```

Comment:
Are these equivalent? I'm having a hard time following the diff here.

Reply:
I believe so. Now that struct-provider syntax is disabled, I had to refactor this. Since I wasn't able to find anywhere where `instrumented_files` or `java` are used (and all the tests passed without them), I opted to remove them.